Reply/app/src/main/java/com/example/reply/ui/MainActivity.kt

```diff
@@ -78,7 +78,16 @@ class MainActivity : ComponentActivity() {
                 val windowSize = calculateWindowSizeClass(this)
                 val devicePosture = devicePostureFlow.collectAsState().value
                 val uiState = viewModel.uiState.collectAsState().value
-                ReplyApp(windowSize.widthSizeClass, devicePosture, uiState)
+                ReplyApp(
+                    windowSize.widthSizeClass,
+                    devicePosture,
+                    uiState,
+                    closeDetailScreen = {
+                        viewModel.closeDetailScreen()
+                    }
+                ) { emailId ->
+                    viewModel.setSelectedEmail(emailId)
+                }
```

Comment:
Very tiny nit, the trailing lambda here looks a little weird since it's another callback instead of displaying content. Could consider moving it into a named parameter like `closeDetailScreen`?